tangelo/linq/helpers/circuits/statevector.py

```diff
@@ -51,27 +51,31 @@ def initializing_circuit(self, return_phase=False):
 
         Args:
             return_phase (bool): Return the global phase that is not captured by the circuit
+            set_n_qubits (bool): Set n_qubits in Circuit to qubits needed for the length of the coefficients array.
 
         Returns:
-            Circuit: The circuit that generates the statevector defined in coeffs
-            float: If return_phase=True, the global phase angle not captured by the Circuit
+            Circuit: The circuit that generates the statevector defined in coeffs, (optional)
```

Comment:
We are not always consistent in our codebase but I'd rather we try:

Can we please start the part after the colon with "Optional, (...)" instead ?